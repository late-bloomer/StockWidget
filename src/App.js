import { styled } from "@mui/material/styles";
import Box from "@mui/material/Box";
import Paper from "@mui/material/Paper";
import Grid from "@mui/material/Grid";
import "./App.css";
import StockPickerWidget from "./Components/StockPickerWidget/StockPickerWidget";
import Header from "./Components/Header/Header";

const Item = styled(Paper)(({ theme }) => ({
  ...theme.typography.body2,
  color: theme.palette.text.secondary,
}));

function App() {
  return (
    <Box sx={{ flexGrow: 1 }}>
      <Grid container>
        <Grid item md={12} lg={12} xl={12} sm={12} xs={12}>
          <Item>
            <Header />
          </Item>
        </Grid>
        <Grid item md={12} lg={12} xl={12} sm={12} xs={12}>
          <Item>
            <StockPickerWidget />
          </Item>
        </Grid>
      </Grid>
    </Box>
  );
}

export default App;
