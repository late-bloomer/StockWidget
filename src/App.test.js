import React from "react";
import ReactDOM from "react-dom";
import Enzyme, { shallow, render, mount } from "enzyme";
import App from "./App";
import toJson from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("App Component", () => {
  it("should render App Successfully", () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
