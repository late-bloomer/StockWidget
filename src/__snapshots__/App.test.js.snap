// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render App Successfully 1`] = `
<ForwardRef(Box)
  sx={
    Object {
      "flexGrow": 1,
    }
  }
>
  <ForwardRef(Grid)
    container={true}
  >
    <ForwardRef(Grid)
      item={true}
      lg={12}
      md={12}
      sm={12}
      xl={12}
      xs={12}
    >
      <Styled(ForwardRef(Paper))>
        <Header />
      </Styled(ForwardRef(Paper))>
    </ForwardRef(Grid)>
    <ForwardRef(Grid)
      item={true}
      lg={12}
      md={12}
      sm={12}
      xl={12}
      xs={12}
    >
      <Styled(ForwardRef(Paper))>
        <WithStyles(StockPickerWidget) />
      </Styled(ForwardRef(Paper))>
    </ForwardRef(Grid)>
  </ForwardRef(Grid)>
</ForwardRef(Box)>
`;

exports[`should render App Successfully 1`] = `
<ForwardRef(Box)
  sx={
    Object {
      "flexGrow": 1,
    }
  }
>
  <ForwardRef(Grid)
    container={true}
  >
    <ForwardRef(Grid)
      item={true}
      lg={12}
      md={12}
      sm={12}
      xl={12}
      xs={12}
    >
      <Styled(ForwardRef(Paper))>
        <Header />
      </Styled(ForwardRef(Paper))>
    </ForwardRef(Grid)>
    <ForwardRef(Grid)
      item={true}
      lg={12}
      md={12}
      sm={12}
      xl={12}
      xs={12}
    >
      <Styled(ForwardRef(Paper))>
        <WithStyles(StockPickerWidget) />
      </Styled(ForwardRef(Paper))>
    </ForwardRef(Grid)>
  </ForwardRef(Grid)>
</ForwardRef(Box)>
`;
